parquet-column/src/test/java/org/apache/parquet/column/page/mem/MemPageWriter.java

```diff
@@ -110,4 +110,13 @@ public String memUsageString(String prefix) {
 
   }
 
+  @Override
+  public void reset() {
+  }
+
+  @Override
+  public void close() {
+  }
```

Comment:
these were unused completely, so I deleted them from the interface on the base class.